docs/resources.rst

```diff
@@ -140,6 +145,9 @@ Solidity Tools
 * `Universal Mutator <https://github.com/agroce/universalmutator>`_
     A tool for mutation generation, with configurable rules and support for Solidity and Vyper.
 
+* `ethdebug <https://github.com/ethdebug/format>`_
```

Comment:
This one should also be moved up to maintain alphabetical order.

Reply:
Moved it up